src/applications/vaos/containers/TypeOfCarePage.jsx

```diff
@@ -52,7 +53,9 @@ export class TypeOfCarePage extends React.Component {
     // kick off the past appointments fetch, which takes a while
     // This could get called multiple times, but the function is memoized
     // and returns the previous promise if it eixsts
-    getLongTermAppointmentHistory();
+    if (this.props.showDirectScheduling) {
```

Comment:
I saw this in the Sentry logs, we're still fetching appt history for the clinics call, but we'll never use it if DS is off